### PR TITLE
Repartition iocs

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Review by fernandohds564 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

File: images/applications/si-ps-trims-m12-ia16t20.bash [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+/usr/local/bin/sirius-ioc-as-ps.py IA-16RaCtrl:CO-PSCtrl-SI1 &> /ioc-logs/sirius-ioc-si-ps-trims-qs-m12-ia16 & # 35.0%
+/usr/local/bin/sirius-ioc-as-ps.py IA-17RaCtrl:CO-PSCtrl-SI1 &> /ioc-logs/sirius-ioc-si-ps-trims-qs-m12-ia17 & # 35.0%
+/usr/local/bin/sirius-ioc-as-ps.py IA-18RaCtrl:CO-PSCtrl-SI1 &> /ioc-logs/sirius-ioc-si-ps-trims-qs-m12-ia18 & # 35.0%
+/usr/local/bin/sirius-ioc-as-ps.py IA-19RaCtrl:CO-PSCtrl-SI1 &> /ioc-logs/sirius-ioc-si-ps-trims-qs-m12-ia19 & # 35.0%
+/usr/local/bin/sirius-ioc-as-ps.py IA-20RaCtrl:CO-PSCtrl-SI1 &> /ioc-logs/sirius-ioc-si-ps-trims-qs-m12-ia20 & # 35.0%
+
+# Diag
+/usr/local/bin/sirius-ioc-as-ps-diag.py SI "(1[1-6]|20)M(1|2)" "Q.*" &> /ioc-logs/sirius-ioc-si-ps-diag-trims-m12-ia16t20 & # 8.6%
```
Comment (anacso17) [inline images/applications/si-ps-trims-m12-ia16t20.bash:10]: the right filter here is: SI "(1[**6-9**]|20)M(1|2)" "Q.*"

**Review by murilobalves — APPROVED** [review]

File: images/applications/ts-ps.bash [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 # Quadrupoles
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS2 &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-12 &
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS04:CO-PSCtrl-TS &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-34 &
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS2 &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-12 & # 13.6%
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS04:CO-PSCtrl-TS &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-34 & # 13.9%
 
 # Correctors
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaCtrl:CO-PSCtrl-TS &> /ioc-logs/env-sirius-ioc-ts-ps-correctors &
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaCtrl:CO-PSCtrl-TS &> /ioc-logs/env-sirius-ioc-ts-ps-correctors & # 40.2%
```
Comment (anacso17) [inline images/applications/ts-ps.bash:12]: Is this log file correct? wouldn't it be `sirius-ioc-ts-ps-correctors`?

Comment (xresende) [reply images/applications/ts-ps.bash]: it is. will be fixed in my next commit

**Comment by fernandohds564** [comment]
@xresende, I don't understand. Is this deprecated because of #7?

**Comment by xresende** [comment]
> @xresende, I don't understand. Is this deprecated because of #7?

no, in the sense that it is useful because it has only changes related to IOC repartition and thus, it is easier to analyze. This is why I kept it. yes, in the sense that if you analyze the other PR completely you can disregard this one.

Ideally you could analyze this one, I merge it, and then you analyze the other PR, which will have only changes related to composing services.

the other PR was created on top of this one, not on master, by mistake. this is the source of the confusion.

**Review by anacso17 — APPROVED** [review]

**Pull request merged**